CONTRIBUTING.md

```diff
@@ -51,7 +55,7 @@ Make sure you've signed the repository's Contributor License Agreement. We are n
  
 4. Submit a pull request
 
-Push your local changes to you forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
+Push local changes to you forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
```

Comment:
```suggestion
Push local changes to your forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
```

Reply:
Is the idea here that a commit subject line should be reused as a PR title? (I don't see any mention of PR titling in that doc...)